### DISCUSSION TITLE
New exceptional SDXL models for Canny, Openpose, and Scribble - [HF download - Trained by Xinsir - h/t Reddit]

### DISCUSSION THREAD

**Discussion by CCpt5** [Mikubill/sd-webui-controlnet#2939] [General]
Just a heads up that these 3 new SDXL models are outstanding. (Searched and didn't see the URL). Caddying this over from Reddit: 

New on June 26, 2024:
[Tile](https://huggingface.co/xinsir/controlnet-tile-sdxl-1.0)
[Depth](https://huggingface.co/xinsir/controlnet-depth-sdxl-1.0)

[Canny](https://huggingface.co/xinsir/controlnet-canny-sdxl-1.0)
[Openpose](https://huggingface.co/xinsir/controlnet-openpose-sdxl-1.0)
[Scribble](https://huggingface.co/xinsir/controlnet-scribble-sdxl-1.0)
[Scribble-Anime](https://huggingface.co/xinsir/anime-painter)

[Xinsir main profile on Huggingface](https://huggingface.co/xinsir)
[Reddit Comments](https://old.reddit.com/r/StableDiffusion/comments/1d24igd/new_sdxl_controlnets_canny_scribble_openpose/)

**Comment by huchenlei** [comment]
Thanks! Added them to the model download wiki page.

**Reply by DavideAlidosi** [reply]
I would love to be able to use especially the new OpenPose model on A1111. I followed and converted the diffuser with this guide, https://github.com/huggingface/diffusers/issues/3015, but without success, I get the message: `WARNING - Unable to determine version for ControlNet model 'openpose [f87f6101]'`. Any tips?
Thank you.

**Reply by huchenlei** [reply]
You just need to rename model to include "xl" in the model file name to avoid this warning.

**Comment by CCpt5** [comment]
@huchenlei Xinsir added a new SDXL Tile and Depth model today: https://huggingface.co/xinsir

**Comment by dilectiogames** [comment]
How this works? They all have the same name, do i download all of them and rename them? And what name do I use.

I have a year of "experience" with this AI stuff but man they do like to make this confusing as hell and throw any knowledge we have out of the window each month

**Reply by AppleJack0012** [reply]
If you have a "year" with this, then you should know what to do with controlnet files from Huggingface. I learned what to do with these files the first month of starting with ai - 3 years ago.

**Reply by dilectiogames** [reply]
> If you have a "year" with this, then you should know what to do with controlnet files from Huggingface. I learned what to do with these files the first month of starting with ai - 3 years ago.

why not just comment with an answer instead of wasting our time bragging how good you are and how fast you learn?

**Reply by AppleJack0012** [reply]
You're hilarious. Contradicting oneself is not a form of flattery. 
<img width="988" height="152" alt="image" src="https://github.com/user-attachments/assets/85d8a257-8573-4539-b831-2796bd1603f8" />
